### DISCUSSION TITLE
[Feature request] Categorise Toots for lists

### DISCUSSION THREAD

**Discussion by 1Syler** [mastodon/mastodon#21037] [General]
I been trying to find the best way to curate what I see and have been using small lists to do this. An issue that I have with this is, I want to create lists like Technology, Politics, Infosec, etc but the lists end up with lots of toots that don't fit in the category because people toot about a range of subjects.

I would like for categories to be added to toots so that I can add users to lists and only get the category of toots I want, so rather than adding @ user@server to a list, I can add @ category@user@server to a list. By default every toot would be tagged as General, users could create their own categories and would be able to select from a list of their categories when they create a toot.

Thoughts?

**Comment by shaedrich** [comment]
We had a similar discussion about following users only for specific topics: https://github.com/mastodon/mastodon/issues/1208#issuecomment-1890224986

Also possibly interesting: #7692
